src/clientmetricreport/DefaultClientMetricReport.ts

```diff
@@ -329,6 +335,89 @@ export default class DefaultClientMetricReport implements ClientMetricReport {
     }
   }
 
+  /**
+   *  media Stream metrics
+   */
+
+     readonly mediaStreamMetricSpec: {
```

Comment:
Do we need this? We already have video and audio downstream and upstream map defined already.

Reply:
I will change the name and move audio metrics to observable metrics specs.